src/test/resources/pages/instructorFeedbackRankMinMaxChoicesSuccess.html

```diff
@@ -1458,10 +1458,10 @@ <h5 class="col-sm-2">
                 </a>
                 <ul class="dropdown-menu" data-container="body" data-original-title="Choose an option" data-placement="top" data-toggle="tooltip" data-trigger="manual" title="">
                   <li>
-                    <a class="feedback-path-dropdown-option" data-giver-type="SELF" data-path-description="Feedback session creator (i.e., me) will give feedback on <span class='glyphicon glyphicon-arrow-right'></span> Nobody specific (For general class feedback)" data-recipient-type="NONE" href="javascript:;" style="display: none;">
+                    <a class="feedback-path-dropdown-option" data-giver-type="SELF" data-path-description="Feedback session creator (i.e., me) will give feedback on <span class='glyphicon glyphicon-arrow-right'></span> Nobody specific (For general class feedback)" data-recipient-type="NONE" href="javascript:;">
```

Comment:
Based on the Travis build log, the changes done to this file (and some others) are all unnecessary, which does make sense since you did not modify any code in this area. I'm guessing you're using some old JS code without rebuilding when fixing this file with God mode.